bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/HydrawiseCloudHandler.java

```diff
@@ -179,6 +180,12 @@ protected void sendStopAllCommand()
         client.stopAllRelays(controllerId);
     }
 
+    private void updateController(Controller controller) {
+        updateGroupState(CHANNEL_GROUP_CONTROLLER, CHANNEL_CONTROLLER_LAST_CONTACT,
+                new DecimalType(controller.lastContact));
```

Comment:
So is this supposed to be a timestamp or the number of seconds since this controller was last contacted?

Reply:
it is, unix time stamp, GMT

Reply:
If it is a timestamp then use a DateTimeType instead.